core/jsoncollector.class.inc.php

```diff
@@ -141,12 +141,13 @@ public function Prepare()
 			} else {
 				$aDataGet = [];
 			}
-			$iSynchroTimeout = (int)Utils::GetConfigurationValue('itop_synchro_timeout', 600); // timeout in seconds, for a synchro to run
-			$aCurlOptions = Utils::GetCurlOptions($iSynchroTimeout);
+			
+			$aCurlOptions = static::GetCurlOptions();
+			
 
 			//logs
 			Utils::Log(LOG_DEBUG, 'Source aDataGet: '.json_encode($aDataGet));
-			$this->sFileJson = Utils::DoPostRequest($this->sURL, $aDataGet, '', $aResponseHeaders, $aCurlOptions);
+			$this->sFileJson = Utils::DoPostRequest($this->sURL, $aDataGet, static::GetOptionalHeaders(), $aResponseHeaders, $aCurlOptions);
```

Comment:
I would suggest to not call it statically, but with the object instance:
```suggestion
			$this->sFileJson = Utils::DoPostRequest($this->sURL, $aDataGet, $this->GetOptionalHeaders(), $aResponseHeaders, $aCurlOptions);
```